target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_arcadyan_vrv9510kwac23.dts

```diff
@@ -0,0 +1,356 @@
+
+/ {
+	compatible = "arcadyan,vrv9510kwac23", "lantiq,xway", "lantiq,vr9";
+	model = "Arcadyan VRV9510KWAC23";
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 mem=254M vpe1_load_addr=0x8fe00000 vpe1_mem=2M maxvpes=1 maxtcs=1 nosmp";
+	};
+
+	aliases {
+		led-boot = &power_green;
+		led-failsafe = &upgrade;
+		led-running = &power_green;
+		led-upgrade = &power_green;
+		led-internet = &internet_green;
+		led-wifi = &wireless_green;
+	};
+
+	bcm43222-sprom {
+		compatible = "brcm,ssb-sprom";
+
+		pci-bus = <0>;
+		pci-dev = <14>;
+
+		nvmem-cells = <&macaddr_boardconfig_16 1>;
+		nvmem-cell-names = "mac-address";
+
+		brcm,sprom = "brcm/bcm43222-sprom.bin";
+		brcm,sprom-fixups = <2   0x04d2>, 
+				    <65  0x1308>, <68  0x0402>,
+				    <70  0x1cc6>, <71  0x3c49>,
+				    <72  0x1132>, <87  0x0315>,
+				    <88  0x0315>, <96  0x2048>,
+				    <97  0xfeed>, <98  0x153e>,
+				    <99  0xfb1f>, <100 0x3e54>,
+				    <101 0x3848>, <102 0xfea0>, 
+				    <103 0x145c>, <104 0xfaf0>,
+				    <105 0xfe6e>, <106 0x110c>,
+				    <107 0xfb7e>, <108 0xff00>,
+				    <109 0x13c4>, <110 0xfb30>, 
+				    <111 0x0000>, <112 0x204c>, 
+				    <113 0xfeb8>, <114 0x1508>, 
+				    <115 0xfacb>, <116 0x3e48>,
+				    <117 0x3848>, <118 0xfeb2>, 
+				    <119 0x156e>, <120 0xfabf>, 
+				    <121 0xfe57>, <122 0x1139>,
+				    <123 0xfb6d>, <124 0xff38>,
+				    <125 0x13ee>, <126 0xfb6d>,
+				    <127 0x0000>, <161 0x0000>,
+				    <162 0x0000>, <169 0x0000>,
+				    <170 0x0000>, <171 0x0000>,
+				    <172 0x0000>, <173 0x0000>,
+				    <174 0x0000>, <175 0x0000>,
+				    <176 0x0000>, <219 0x1108>,
+				    <127 0x0000>, <161 0x0000>,
+				    <162 0x0000>, <169 0x0000>,
+				    <170 0x0000>, <171 0x0000>,
+				    <172 0x0000>, <173 0x0000>,
+				    <174 0x0000>, <175 0x0000>,
+				    <176 0x0000>, <219 0x1108>;
+	};
+
+	bcm4360-sprom {
+		compatible = "brcm,bcma-sprom";
+
+		pci-bus = <2>;
+		pci-dev = <0>;
+
+		nvmem-cells = <&macaddr_boardconfig_16 1>;
+		nvmem-cell-names = "mac-address";
+
+		brcm,sprom = "brcm/bcm4360-sprom.bin";
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 20 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+		wps {
+			label = "wps";
+			gpios = <&gpio 6 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+		rfkill {
+			label = "rfkill";
+			gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RFKILL>;
+		};
+		service {
+			label = "service";
+			gpios = <&gpio 22 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_0>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		// from top to bottom
+		power_green: power {
+			label = "green:power";
+			gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+			default-state = "keep";
+		};
+		internet_green: internet_green {
+			label = "green:internet";
+			gpios = <&gpio 26 GPIO_ACTIVE_LOW>;
```

Comment:
Please replace the `label` attribute in the LED definition with the new definition:
For this LED it would look like this:
````
		internet_green: internet_green {
			gpios = <&gpio 26 GPIO_ACTIVE_LOW>;
			color = <LED_COLOR_ID_GREEN>;
			function = LED_FUNCTION_WAN;
		}
````

Reply:
Done